tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -107,11 +130,57 @@ def test_multiprocess(storage_url: str) -> None:
     study_name = _STUDY_NAME
     optuna.create_study(storage=storage_url, study_name=study_name)
     with Pool(n_workers) as pool:
-        pool.map(run_optimize, [(study_name, storage_url)] * n_workers)
+        pool.map(run_optimize, [(study_name, storage_url, 1, False)] * n_workers)
 
     study = optuna.load_study(study_name=study_name, storage=storage_url)
 
     trials = study.trials
     assert len(trials) == n_workers * 20
 
     _check_trials(trials)
+
+
+def test_multiprocess_with_progbar(
+    capsys: _pytest.capture.CaptureFixture, storage_url: str
+) -> None:
+    with capsys.disabled():
+        n_workers = 8
+        study_name = _STUDY_NAME
+        optuna.create_study(storage=storage_url, study_name=study_name)
+        with Pool(n_workers) as pool:
+            ret = pool.map(run_optimize, [(study_name, storage_url, 1, True)] * n_workers)
+            comp_cnt = 0
+            for i in range(n_workers):
+                if "20/20" in ret[i] and "100%" in ret[i]:
+                    comp_cnt += 1
+            assert comp_cnt == n_workers
+
+        study = optuna.load_study(study_name=study_name, storage=storage_url)
+
+        trials = study.trials
+        assert len(trials) == n_workers * 20
+
+        _check_trials(trials)
+
+
+def test_multiprocess_with_progbar_parallel(
```

Comment:
We can merge this function to `test_multiprocess_with_progbar` with `parametrize` decorator.